fortitude/src/check.rs

```diff
@@ -119,6 +120,7 @@ pub struct CheckSettings {
     pub line_length: usize,
     pub file_extensions: Vec<String>,
     pub output_format: OutputFormat,
+    pub no_progress_bar: bool,
```

Comment:
If we want to be really rusty (rustic?), this should probably be a new enum:

```rust
#[derive(Debug, Clone, Copy, PartialEq, Eq, Default, CacheKey, is_macro::Is)]
pub enum ProgressBar {
    #[default]
    Disabled,
    Enabled,
}

impl From<bool> for ProgressBar {
    fn from(version: bool) -> Self {
        if version {
            ProgressBar::Enabled
        } else {
            ProgressBar::Disabled
        }
    }
}

impl Display for ProgressBar {
    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
        match self {
            Self::Disabled => write!(f, "disabled"),
            Self::Enabled => write!(f, "enabled"),
        }
    }
}
```

(just copy-pasting from `PreviewMode`)

This just gives type-safety if passing this around -- can't accidentally pass it to `PreviewMode` for example!

Reply:
I've now done this, but went with three options: `Off`, `Ascii`, and `Fancy`.